main.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'app'
+
+def display_menu
+  puts 'Please choose an option by entering a number: '
+  CHOICES.each { |key, value| puts "#{key} - #{value.to_s.tr('_', ' ')}" }
+  gets.chomp.to_i
+end
+
+def handle_option(app, option)
+  choice = CHOICES[option]
+  if choice
+    app.send(choice)
```

Comment:
- [ ] If I ignored the above mistake, to check your `menu` functionality based on what user gives input then, I found that you are calling the `send` method for the `app` class, which is also doesn't exist at all. Please make sure that you didn't missed anything at your end while committing the changes on the branch. Because I wasn't able to check your `menu` functionality as a first reviewer, your second reviewer will thoroughly make review on your menu functionality to ensure application integrity and working as per guidelines. Good Luck 👍🏻